airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/IntegrationConfig.java

```diff
@@ -68,6 +77,14 @@ public Optional<Path> getStatePath() {
     return Optional.ofNullable(statePath);
   }
 
+  static class ConfigValidationException extends Exception {
+
```

Comment:
nit: what's with the extra empty lines?

Reply:
good question, I think it was auto formatted?